src/Libraries/RevitNodes/Elements/LinkElement.cs

```diff
@@ -0,0 +1,485 @@
+﻿using Autodesk.Revit.DB;
+using System;
+using Revit.Elements;
+using System.Collections.Generic;
+using System.Linq;
+using DynamoUnits;
+using Dynamo.Graph.Nodes;
+using View = Revit.Elements.Views.View;
+using System.Text;
+using System.Threading.Tasks;
+using Level = Revit.Elements.Level;
+using RevitServices.Persistence;
+using Autodesk.Revit.UI;
+using System.Windows;
+using Autodesk.DesignScript.Geometry;
+using Revit.GeometryConversion;
+using System.Windows.Media.Imaging;
+using Point = Autodesk.DesignScript.Geometry.Point;
+using Autodesk.Revit.UI;
+using System.Windows.Media.Media3D;
+using Autodesk.DesignScript.Runtime;
+using System.ComponentModel;
+
+namespace Revit.Elements
+{
+    /// <summary>
+    /// A Revit Link Instance
+    /// </summary>
+    public static class LinkElement
+    {
+
+        #region Helpers
+        private static List<RevitLinkInstance> GetLinkInstancesContainingLinkElement(Element linkElement)
+        {
+            Autodesk.Revit.DB.Element revitElement = linkElement.InternalElement;
```

Comment:
What if, by accident, the `linkElement` given is not actually a link element, but an element in the current Revit document?